synapse/config/server.py

```diff
@@ -57,6 +57,64 @@
 --------------------------------------------------------------------------------"""
 
 
+KNOWN_LISTENER_TYPES = [
```

Comment:
Not a big deal, but I'd make this and `KNOWN_RESOURCES` each a `set`.

Reply:
makes sense. updated.